### PR TITLE
Sidebar with Fixed bottom buttons

### PR DESCRIPTION
**Pull request by vishwajeetraj11**
https://github.com/marlinprotocol/interface-v2/assets/47270995/3a295587-c899-471e-bdf2-a8a839a6360a

### CODE REVIEW

**Comment by nayanmarlin** [comment]
@vishwajeetraj11 - I think you need to rebase this PR with latest code in latest-ui branch.

**Review by paul4702 — CHANGES_REQUESTED** [review]
Can you please take a pull from the latest UI as there is a file structure change to resolve the conflicts.

File: src/lib/components/v2/header/sub-components/SidebarLinksGroup.svelte [hunk 1]
```diff
@@ -2,11 +2,41 @@
...
 	let links: SidebarLinks[] = [];
+	let checked = false;
+
+	type MenuItemType = {
+		iconSVG: string;
+		label: string;
+	};
+
+	const menuItems: MenuItemType[] = [
+		{
```
Comment (nayanmarlin) [inline src/lib/components/v2/header/sub-components/SidebarLinksGroup.svelte:19]: @vishwajeetraj11 - Can we remove this SVG from list? as i mentioned today's call?

File: src/lib/components/v2/header/sub-components/SidebarLinksGroup.svelte [hunk 2]
```diff
@@ -178,4 +208,104 @@
...
+				<svg
+					width="14"
+					height="14"
+					viewBox="0 0 14 14"
+					fill="none"
+					class="{isNavOpen ? 'col-start-2' : 'col-start-1'} row-start-1"
+					xmlns="http://www.w3.org/2000/svg"
+				>
+					<path
+						opacity="0.4"
```
Comment (nayanmarlin) [inline src/lib/components/v2/header/sub-components/SidebarLinksGroup.svelte:257]: @vishwajeetraj11 - same here

**Review by nayanmarlin — CHANGES_REQUESTED** [review]

**Review by nayanmarlin — APPROVED** [review]

**Pull request merged**